modules/ztp-du-cluster-config-reference.adoc

```diff
@@ -0,0 +1,92 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp-vdu-configuration-reference.adoc
+
+:_module-type: REFERENCE
+[id="ztp-du-cluster-config-reference_{context}"]
+= Recommended cluster configurations to run vDU applications
+
+Clusters running virtualized distributed unit (vDU) applications require a highly tuned and optimized configuration. The following information describes the various elements that you require to support vDU workloads in {product-title} {product-version} clusters.
+
+[NOTE]
+====
+You manage your custom site configuration data in a Git repository accessible from the hub cluster that is defined as a source repository for Argo CD. You extract the baseline source custom resources (CRs) from the zero touch provisioning (ZTP) container.
+====
+
+== Recommended cluster MachineConfig installation CRs
+
+The following `MachineConfig` installation CRs configure the cluster host:
+
+.MachineConfig installation CRs
+[cols=2*, width="80%", options="header"]
+|====
+|CR filename
+|Description
+
+|`02-workload-partitioning.yaml`
+|Configures workload partitioning for the cluster.
```

Comment:
By leaving out "installation" in the title, we should make a note that this MachineConfig must be applied at installation time.